### DISCUSSION TITLE
Can't open mesh file

### DISCUSSION THREAD

**Discussion by CrazyMan-IK** [gildor2/UEViewer#266] [Q&A] [answered]
When trying to open some models from the Poppy Playtime game, the program crashes with an error
![image](https://user-images.githubusercontent.com/47449484/160622499-fea1cbb2-4f7a-4b83-9c68-b13740493c75.png)

**Comment by gildor2** [comment, marked as answer]
https://www.gildor.org/smf/index.php/topic,7955.0.html
Forum says it's UE4.26, you're using UE4.27

**Reply by CrazyMan-IK** [reply]
I checked the .pak file with this method https://gbatemp.net/threads/how-to-unpack-and-repack-unreal-engine-4-files.531784/ and it showed that the file is from version 4.27

Apparently their method is not fully working, thank you very much, now the files are opening :D
